SQLite3: panic: interface conversion: interface {} is nil, not string

Hey, thanks for such a cool project. I'm having an issue while running it on my database, though:
```
$ plant_erd sqlite3 --database navidrome.db

panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/sue445/plant_erd/adapter/sqlite3.(*Adapter).getForeignKeys(0xc0000a8110, {0xc0000acb60?, 0x100512a3c?})
        /Users/runner/work/plant_erd/plant_erd/adapter/sqlite3/adapter.go:115 +0x3eb
github.com/sue445/plant_erd/adapter/sqlite3.(*Adapter).GetTable(0xc0000a8110, {0xc0000acb60, 0xc})
        /Users/runner/work/plant_erd/plant_erd/adapter/sqlite3/adapter.go:80 +0x454
github.com/sue445/plant_erd/lib.LoadSchema({0x1005b1e48, 0xc0000a8110})
        /Users/runner/work/plant_erd/plant_erd/lib/schema.go:17 +0x12c
main.main.func1(0xc00014a160?)
        /Users/runner/work/plant_erd/plant_erd/cmd/plant_erd/main.go:64 +0x86
github.com/urfave/cli.HandleAction({0x1004b0e20?, 0xc0000a2270?}, 0x7?)
        /Users/runner/go/pkg/mod/github.com/urfave/cli@v1.22.10/app.go:524 +0x50
github.com/urfave/cli.Command.Run({{0x100516141, 0x7}, {0x0, 0x0}, {0xc000099910, 0x1, 0x1}, {0x10051f9c8, 0x19}, {0x0, ...}, ...}, ...)
        /Users/runner/go/pkg/mod/github.com/urfave/cli@v1.22.10/command.go:173 +0x65b
github.com/urfave/cli.(*App).Run(0xc000146000, {0xc0000b4000, 0x5, 0x5})
        /Users/runner/go/pkg/mod/github.com/urfave/cli@v1.22.10/app.go:277 +0x8c7
main.main()
        /Users/runner/work/plant_erd/plant_erd/cmd/plant_erd/main.go:213 +0x1132
```

This is the file I'm trying to read (no worries, data is dummy):
[navidrome.db.zip](https://github.com/sue445/plant_erd/files/10996574/navidrome.db.zip)

I'm using macOS amd64 and this happens with both the latest release 0.4.2 and building it from master.

Hopefully this is a simple fix :)

--- a/adapter/sqlite3/adapter.go
+++ b/adapter/sqlite3/adapter.go
@@ -110,9 +110,18 @@ func (a *Adapter) getForeignKeys(tableName string) ([]*db.ForeignKey, error) {
 			return nil, err
 		}
 
+		toColumn := ""
+		if row["to"] == nil {
+			// NOTE: If `to` is NULL, implicitly equals `id` (maybe...)
+			// c.f. https://github.com/diesel-rs/diesel/issues/1535
+			toColumn = "id"
+		} else {
+			toColumn = row["to"].(string)
+		}
+
 		foreignKey := &db.ForeignKey{
 			FromColumn: row["from"].(string),
-			ToColumn:   row["to"].(string),
+			ToColumn:   toColumn,
 			ToTable:    row["table"].(string),
 		}
 

--- a/adapter/sqlite3/adapter_test.go
+++ b/adapter/sqlite3/adapter_test.go
@@ -73,6 +73,18 @@ func TestAdapter_GetTable(t *testing.T) {
 		a.DB.MustExec("CREATE UNIQUE INDEX index_user_id_and_target_user_id_on_followers ON followers(user_id, target_user_id)")
 		a.DB.MustExec("CREATE UNIQUE INDEX index_target_user_id_and_user_id_on_followers ON followers(target_user_id, user_id)")
 
+		a.DB.MustExec(`
+			CREATE TABLE album_genres (
+				album_id varchar default null not null
+					references album
+						on delete cascade,
+				genre_id varchar default null not null
+					references genre
+						on delete cascade,
+				constraint album_genre_ux
+					unique (album_id, genre_id)
+		);`)
+
 		type args struct {
 			tableName string
 		}
@@ -184,6 +196,46 @@ func TestAdapter_GetTable(t *testing.T) {
 						{
 							Name:    "index_user_id_and_target_user_id_on_followers",
 							Columns: []string{"user_id", "target_user_id"},
+							Unique:  true,
+						},
+					},
+				},
+			},
+			{
+				name: "album_genres",
+				args: args{
+					tableName: "album_genres",
+				},
+				want: &db.Table{
+					Name: "album_genres",
+					Columns: []*db.Column{
+						{
+							Name:    "album_id",
+							Type:    "varchar",
+							NotNull: true,
+						},
+						{
+							Name:    "genre_id",
+							Type:    "varchar",
+							NotNull: true,
+						},
+					},
+					ForeignKeys: []*db.ForeignKey{
+						{
+							FromColumn: "genre_id",
+							ToTable:    "genre",
+							ToColumn:   "id",
+						},
+						{
+							FromColumn: "album_id",
+							ToTable:    "album",
+							ToColumn:   "id",
+						},
+					},
+					Indexes: []*db.Index{
+						{
+							Name:    "sqlite_autoindex_album_genres_1",
+							Columns: []string{"album_id", "genre_id"},
 							Unique:  true,
 						},
 					},

